### ISSUE TITLE
Example returns _react3.default.createClass is not a function.

### ISSUE THREAD

**Issue by boxxa** [inProgress-team/react-native-meteor#281]
Looks like createContainer.js:21:27 is causing the error on startup.

`import React, { Component } from 'react';
import { View, Text } from 'react-native';

import Meteor, { createContainer } from 'react-native-meteor';
Meteor.connect('ws://localhost:3000/websocket');

class App extends Component {
  render() {
    const { usersReady } = this.props;
    return (
      <View>
          <Text>Hello</Text>
            {!usersReady && <Text>Connecting...</Text>}
        </View>
    );
  }
}

export default createContainer(params=>{
  const handle = Meteor.subscribe('users');

  return {
    usersReady: handle.ready(),
  };
}, App)`

**Comment by blackjack0v0** [comment]
**React.createClass  has been duplicated after react v15.5.0**, it was isolated to a package called 
create-react-class, so you  should install it 
`npm install  create-react-class --save `
**then modify the  createContainer.js  inside node_modules/react-native-meteor/src/component/**
`import createReactClass from 'create-react-class';`
  ```
....
return createReactClass({
    displayName: 'MeteorDataContainer',
    mixins: [Mixin],
.....
```

**Comment by blackjack0v0** [comment]
hope this probrem fixed in next update

**Comment by smooJitter** [comment]
@blackjack0v0 this works until I run yarn add .... again.  Running yarn seems to restore the original version.

**Comment by lc3t35** [comment]
Hi, can we expect an update soon fixing last issues ? do you need help ?

**Comment by charpeni** [comment]
Fixed in `1.2.0`.
